tests/Support/SupportArrTest.php

```diff
@@ -4,6 +4,7 @@
 
 use stdClass;
 use ArrayObject;
+use Carbon\Carbon;
```

Comment:
We enforce `\Illuminate\Support\Carbon` since https://github.com/laravel/framework/pull/19771